renderer/locales/id.json

```diff
@@ -0,0 +1,288 @@
+      "START_BUTTON_TITLE": "Upscayl 🚀",
+      "IN_PROGRESS_BUTTON_TITLE": "Upscayling ⏳"
+    },
+    "MORE_OPTIONS_DRAWER": {
+      "RESET_BUTTON_TITLE": "Reset Gambar",
+      "LENS_VIEW_TITLE": "Tampilan Lensa",
+      "SLIDER_VIEW_TITLE": "Tampilan Slider",
+      "ZOOM_AMOUNT_TITLE": "Jumlah Pembesaran",
+      "TOTAL_UPSCAYLS": "Total Upscayl",
+      "TOTAL_BATCH_UPSCAYLS": "Total Batch Upscayl",
+      "TOTAL_IMAGE_UPSCAYLS": "Total Upscayl Gambar",
+      "TOTAL_DOUBLE_UPSCAYLS": "Total Double Upscayl",
+      "AVERAGE_UPSCAYL_TIME": "Waktu Rata-rata Upscayl",
+      "LAST_UPSCAYL_DURATION": "Durasi Upscayl Terakhir",
+      "LAST_USED_AT": "Terakhir Digunakan Pada"
+    },
+    "PROGRESS_BAR": {
+      "BATCH_UPSCAYL_IN_PROGRESS_TITLE": "Batch Upscayl Sedang Berlangsung:",
+      "IN_PROGRESS_TITLE": "Melakukan sihir Upscayl...",
+      "STOP_BUTTON_TITLE": "STOP"
+    },
+    "RESET_BUTTON_TITLE": "Reset",
+    "RIGHT_PANE_INFO": {
+      "SELECT_FOLDER": "Pilih Folder",
+      "SELECT_IMAGE": "Pilih Gambar",
+      "SELECT_FOLDER_DESCRIPTION": "Pastikan folder tidak berisi apapun kecuali gambar PNG, JPG, JPEG & WEBP.",
+      "SELECT_IMAGES_DESCRIPTION": "Pilih atau seret dan lepas gambar PNG, JPG, JPEG atau WEBP.",
+      "PASTE_IMAGE_DESCRIPTION": "Tekan Ctrl+V atau ⌘+V untuk Menempel gambar dari Clipboard"
+    },
+    "PROGRESS": {
+      "PROCESSING_TITLE": "Memproses gambar...",
+      "SCALING_CONVERTING_TITLE": "Memperbesar dan mengonversi gambar...",
+      "WAIT_TITLE": "Tunggu sebentar...",
+      "SUCCESS_TITLE": "Upscayl Berhasil!",
+      "BATCH": {
+        "SELECTED_FOLDER_TITLE": "Folder yang dipilih:",
+        "DONE_TITLE": "Semua selesai!",
+        "OPEN_UPSCAYLED_FOLDER_TITLE": "Buka Folder Upscayl"
+      }
+    },
+    "SLIDER": {
+      "ORIGINAL_TITLE": "Asli",
+      "UPSCAYLED_TITLE": "Upscayl"
+    },
+    "DIALOG_BOX": {
+      "CLOSE": "Tutup"
+    }
+  },
+  "ERRORS": {
+    "GPU_ERROR": {
+      "TITLE": "Kesalahan GPU",
+      "DESCRIPTION": "Mengalami masalah dengan GPU. Silakan baca dokumen untuk pemecahan masalah! ({data})"
+    },
+    "COPY_ERROR": {
+      "TITLE": "Kesalahan Salin",
+      "DESCRIPTION": ""
+    },
+    "READ_WRITE_ERROR": {
+      "TITLE": "Kesalahan Baca/Tulis",
+      "DESCRIPTION": "Pastikan lokasi benar dan Anda memiliki izin baca/tulis yang tepat \n({data})"
+    },
+    "TILE_SIZE_ERROR": {
+      "TITLE": "Kesalahan",
+      "DESCRIPTION": "Ukuran tile salah. Silakan ubah ukuran tile di pengaturan atau setel ke 0 ({data})"
+    },
+    "EXCEPTION_ERROR": {
+      "TITLE": "Kesalahan Sistem",
+      "DESCRIPTION": "Upscayl mengalami kesalahan. Mungkin, binari upscayl gagal mengeksekusi perintah dengan benar. Coba periksa log untuk melihat apakah Anda mendapatkan informasi. Anda dapat memposting masalah di repositori GitHub Upscayl untuk bantuan lebih lanjut."
+    },
+    "GENERIC_ERROR": {
+      "TITLE": "Kesalahan"
```

Comment:
Please use 'Galat' instead of 'Kesalahan' to all 'error' strings.

Reply:
Thank you for the suggestion! While 'galat' is a valid term, it feels more formal than 'kesalahan,' which aligns better with the app's tone.

Reply:
No worry, now 'galat' commonly used to refer 'error'.

Reply:
Thank you for the clarification! I'll update the strings to use 'galat' to ensure consistency and align with common usage.